core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/GenerateDot.scala

```diff
@@ -46,6 +46,7 @@ import org.apache.spark.sql.rapids.tool.profiling.{ApplicationInfo, SparkPlanInf
 object GenerateDot {
   val GPU_COLOR = "#76b900" // NVIDIA Green
   val CPU_COLOR = "#0071c5"
+  // TODO: Add color for Photon nodes
```

Comment:
not sure why this is here?  Are we printing the photon plan or the converted plan or do we have options for both?

Reply:
Removed this